sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -8,12 +8,18 @@
 
 #pragma once
 
+#include "azure/core/platform.hpp"
+
 #include <cstring>
 #include <new> // for placement new
 #include <random>
 #include <string>
 #include <utility> // for swap and move
 
+#if defined(AZ_PLATFORM_POSIX)
+#include <openssl/rand.h> //for RAND_bytes
```

Comment:
I believe this is introducing a dependency to the SDK we didn't previously have. Will the customer always have this header available, even if the user doesn't install `curl` from vcpkg (since it's optional)?

Do we need to update getting started?

Reply:
openssl isn't a new dependency.  It already exists for MD5.

Reply:
Ah dang it, you are right, good point.